docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -0,0 +1,75 @@
+# macOS setup
+
+In Fleet you can customize the first time macOS setup experience for your end users.
+
+You can require end users to authenticate with your identity provider (IdP) and agree to and end user license agreement (EULA) before they can use their new Mac.
+
+You can customize the macOS Setup Assistant and choose to show or hide specific panes.
+
+Also, you can bootstrap new Macs with your configuration management tool of choice (ex. Munki, Chef, or Puppet).
```

Comment:
I think we should be very literal. Strictly speaking, this is not the only thing we are enabling, right? We are allowing you to run an script at set up time. As part of this, you can install an agent, you can install a million other apps, or you can auto-popup a rickroll.

Reply:
Hey @zhumo I updated this sentence to communicate that bootstrap package allows for "full control over the setup experience by installing tools like Puppet, Munki, DEP notify, custom scrips, and more."

What do you think?